docs/Common/Enums/index.md

```diff
@@ -889,6 +889,7 @@
 |VEHICLEDATA_ELECTRONICPARKBRAKESTATUS|29||
 |VEHICLEDATA_CLOUDAPPVEHICLEID|30| Parameter used by cloud apps or the policy server to identify a head unit|
 |VEHICLEDATA_OEM_CUSTOM_DATA|31||
+|VEHICLEDATA_GEARSTATUS|32||
```

Comment:
The enum value should match the position in the HMI_API spec https://github.com/LuxoftSDL/sdl_core/blob/feature/0266_new_vehicle_data_gear_status/src/components/interfaces/HMI_API.xml#L1174

Gear Status should be placed before PRNDL
```md
|VEHICLEDATA_GEARSTATUS|8||
```
and the subsequent enum values should be incremented.
```md
|VEHICLEDATA_PRNDL|9||
|VEHICLEDATA_TIREPRESSURE|10||
|VEHICLEDATA_ODOMETER|11||
|VEHICLEDATA_BELTSTATUS|12||
...
```

Reply:
fixed in f87bcddf47dfcb1763618e5d8fdadfd8a17966b5